system_tests/system_test.go

```diff
@@ -338,7 +338,7 @@ CONSOLE_LOG=new`
 				pvc, err := clientSet.CoreV1().PersistentVolumeClaims(namespace).Get(ctx, pvcName, metav1.GetOptions{})
 				Expect(err).ToNot(HaveOccurred())
 				return pvc.Spec.Resources.Requests["storage"]
-			}, 120, 5).Should(Equal(newCapacity))
+			}, 180, 5).Should(Equal(newCapacity))
```

Comment:
Honestly I'd be tempted to bump these to 5x. Timeouts in automated tests are my bugbear. The chance of them spotting a problem and not being ignored are too low to justify a 'reasonable time'. If this operation takes 10 minutes, are we really that worried? Or would we chalk it up to infrastructure contention?